getting-started/modules/core/valtimo-dependencies.md

```diff
@@ -33,33 +23,36 @@ whether Maven or Gradle is used:
 #### Gradle dependency:
 ```groovy
 dependencies {
-  implementation platform("com.ritense.valtimo:valtimo-dependencies:${valtimo_version}")
-}
-
-dependencyManagement {
-  imports {
-    mavenBom("com.ritense.valtimo:valtimo-dependencies:${valtimo_version}")
-  }
+  implementation("com.ritense.valtimo:valtimo-dependencies")
 }
 ```
 
 ### Dependencies
 
 The following modules are added when a dependency on Valtimo dependencies is added:
 * [Audit](audit.md)
+* [Authorization](authorization.md)
 * [Case](case.md)
+* Changelog
 * [Connector](connector.md)
 * [Contract](contract.md)
+* [Core](core.md)
+* [Dashboard](dashboard.md)
+* Data provider
 * [Document](document.md)
+* [Exporter](exporter.md)
 * [Form](form.md)
-* Form flow
+* [Form flow](form-flow.md)
```

Comment:
Can you split off form-flow and form-flow-valtimo into different pages?

Reply:
done